sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2300,6 +2332,91 @@ def __getitem__(self, key):
     return self.attributes[key].__get__(self, WriteResult)
 
 
+def _default_io_expansion_service(append_args=None):
+  return BeamJarExpansionService(
+      'sdks:java:io:google-cloud-platform:expansion-service:build',
+      append_args=append_args)
+
+
+class StorageWriteToBigQuery(PTransform):
```

Comment:
Can we better integrate this transform into the existing bigquery sink API by adding a WriteMethod ?

https://github.com/apache/beam/blob/39cab94361bc44e8b48257385d37d8978ab7fc02/sdks/python/apache_beam/io/gcp/bigquery.py#L1753

Reply:
Yup, good idea